### PR TITLE
Extra docs on Spoons

### PR DESCRIPTION
**Pull request by dotemacs**
As per my discussion with Habbie@IRC I'm adding this to make the discoverability of Spoons easier.

I'm prompted to add it since I did stumble upon https://www.hammerspoon.org/Spoons/ but wasn't able to find it easily again.

Also added a link to Spoons from the homepage in the navbar.

### CODE REVIEW

File: getting-started.md [hunk 1]
```diff
@@ -74,6 +75,20 @@ hs.hotkey.bind({"cmd", "alt", "ctrl"}, "W", function()
 end)
 ```
 
+### <a name="spoons"></a>Spoons
+
+Spoons are pre made plugins for Hammerspoon.
```
Comment (Habbie) [inline getting-started.md]:
```suggestion
Spoons are premade plugins for Hammerspoon.
```

File: getting-started.md [hunk 2]
```diff
@@ -74,6 +75,20 @@ hs.hotkey.bind({"cmd", "alt", "ctrl"}, "W", function()
...
+
+Have a look at the official documentation for Spoons
+[here](https://github.com/Hammerspoon/hammerspoon/blob/master/SPOONS.md)
+about how they work and how to use them.
+
+The official website with the listing of Spoons is
+[https://www.hammerspoon.org/Spoons/](https://www.hammerspoon.org/Spoons/)
+and the official repository is
+[https://github.com/Hammerspoon/hammerspoon](https://github.com/Hammerspoon/hammerspoon).
```
Comment (Habbie) [inline getting-started.md]: Did you mean https://github.com/Hammerspoon/Spoons here?

File: index.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ What gives Hammerspoon its power is a set of extensions that expose specific pie
 
 You can write Lua code that interacts with OS X APIs for applications, windows, mouse pointers, filesystem objects, audio devices, batteries, screens, low-level keyboard/mouse events, clipboards, location services, wifi, and more.
 
-If you want to explore the options Hammerspoon offers, check out the [Getting Started Guide](/go/) and the [full API documentation](/docs/).
+If you want to explore the options Hammerspoon offers, check out the [Getting Started Guide](/go/) and the [full API documentation](/docs/) as well as the already pre made plugins called [Spoons](https://www.hammerspoon.org/Spoons/).
```
Comment (Habbie) [inline index.md]:
```suggestion
If you want to explore the options Hammerspoon offers, check out the [Getting Started Guide](/go/) and the [full API documentation](/docs/) as well as the already premade plugins called [Spoons](https://www.hammerspoon.org/Spoons/).
```

File: getting-started.md (hunk 2 above)
Comment (dotemacs) [reply getting-started.md]: Ah, yes. Fixing now.

File: getting-started.md (hunk 1 above)
Comment (dotemacs) [reply getting-started.md]: According to Apple's Dictionary, which taps into Oxford English dictionary for British English it's `pre-made`.

Web version of Oxford English redirects from `premade` to [pre-make](https://en.oxforddictionaries.com/definition/pre-make).

According to [Merriam Webster dictionary](https://www.merriam-webster.com/dictionary/premade), it's `premade`.

Oh, the dilema, which dictionary is more authoritative ?

Are you sticking with your original option of a single word, no hyphen? :)

File: getting-started.md (hunk 2 above)
Comment (dotemacs) [reply getting-started.md]: Fixed. I'll mark this as resolved.

**Comment by dotemacs** [comment]
@Habbie also added 4bcd042e134c51671a57b747e1233216e21dcb10 just now, as I don't see the need to create a separate PR just for that little change.

File: getting-started.md (hunk 1 above)
Comment (Habbie) [reply getting-started.md]: I'm happy with a hyphen or nothing. I'm not happy with a space :)

Comment (dotemacs) [reply getting-started.md]: Went with the hyphenated version

File: index.md (hunk 3 above)
Comment (dotemacs) [reply index.md]: Resolved.

**Comment by dotemacs** [comment]
@Habbie to preview:

```
bundle install
bundle exec jekyll serve
```
and then open http://localhost:4000/

**Comment by dotemacs** [comment]
@Habbie a gentle ping...

**Pull request merged**